src/components-examples/angular/tabs/tab-group-async/tab-group-async-example.html

```diff
@@ -0,0 +1,8 @@
+<ng-container *ngIf="(asyncTabs | async) === null"> Loading tabs... </ng-container>
```

Comment:
I don't really see the benefit of this example, it's not tabs related, right? It's up to the user on how he fills data in.

Reply:
That is correct. My counterquestion is; Does it hurt?

Reply:
Yes a little bit :-) No, I just thought it's also a little bit weird with this 1 second delay at page load.